selfdrive/ui/qt/sidebar.cc

```diff
@@ -47,17 +48,34 @@ Sidebar::Sidebar(QWidget *parent) : QFrame(parent) {
   pm = std::make_unique<PubMaster, const std::initializer_list<const char *>>({"userFlag"});
 }
 
+void Sidebar::mousePressEvent(QMouseEvent *event) {
+  if (onroad && home_btn.contains(event->pos())) {
+    flag_pressed = true;
```

Comment:
```suggestion
    bookmark_pressed = true;
```

Reply:
we'll keep it as flag in all of the code for now